_includes/partners/header.html

```diff
@@ -1,12 +1,10 @@
 <section class="login-developers-links">
   <div class="container">
     <div class="grid-col display-flex flex-justify-end flex-align-center">
-      <a
-        class="usa-nav__link usa-link"
-        href="/"
```

Comment:
Could we just ?

```html
href="{{ site.baseurl }}/"
```

Reply:
GIANT FACEPALM yes that makes way more sense

Reply:
fixed in 47494309aae0bc82812121c7af7910c83a8ac315